ETWAnalyzer/EventDump/DumpFile.cs

```diff
@@ -257,6 +269,13 @@ void PrintData(List<MatchData> aggregatedByDirectory)
                     if (IsPerProcess && bPrintOnce && !IsTotalMode)
                     {
                         ColorConsole.WriteEmbeddedColorLine($"[grey]{group.Key.GetProcessWithId(UsePrettyProcessName)}{GetProcessTags(group.Key, group.First().SessionStart)}[/grey] {(NoCmdLine ? "" : group.Key.CmdLine)}", ConsoleColor.DarkCyan);
+                        
+                        ModuleDefinition fileModule = group.First().FileModule;
+                        string moduleInfo = fileModule != null ? GetModuleString(fileModule, true) : "";
+                        if (moduleInfo!="")
+                        {
+                            ColorConsole.WriteEmbeddedColorLine($"[red]{moduleInfo}[/red]");
```

Comment:
this would print the module info on an extra line. Like the other commands it should be on the same line as the command line to stay consistent. One can use -NoCmdLine to omit the command line to sanitize the output if necessary.

Reply:
resolved